README.md

```diff
@@ -53,11 +58,11 @@ resource "logdna_view" "my_view" {
 
 Using the logdna_view resource- a user can create a view with a provided name query, hosts, category, tags, email, pagerduty and webhook channels, delete a view with a given viewid or update a view's properties.
```

Comment:
```suggestion
Using the `logdna_view` resource, a user can create a View with provided `name`, `query`, `hosts`, `categories`, `tags`, `email_channel`, `pagerduty_channel`, and `webhook_channel`; delete a View with a given `viewid`; or update a View's properties.
```

Also, should they also have to create `apps` values when creating a View?

Reply:
not sure if I understand the create quesiton @nimbinatus. Are you asking whether the app should already exist?

Reply:
also these suggestions are helpful thank you :)

Reply:
The example configuration shows an `apps` key with relevant values [here](https://github.com/logdna/terraform-provider-logdna/pull/4/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R21). My initial thought on reading that configuration and then seeing all of the other keys given for creating a View is that the `apps` key is missing in the list. Should someone creating a View need to provide an `apps` value? The same question applies for the `levels` key. It's not clear which values are required and which ones are optional.

Reply:
@nimbinatus we've documented which fields are required and optional in the terraform docs here: https://github.com/logdna/terraform-provider-logdna/blob/main/docs/resources/logdna_view.md. Is that something you would like to see in the README.md as well?